modules/genomic_browser/ajax/genomic_file_upload.php

```diff
@@ -107,6 +113,46 @@ function validateRequest()
 }
 
 /**
+ * Sets $fileToUpload->full_path
+ *
+ * @param stdClass $fileToUpload The file to upload
+ *
+ * @return void
+ */
+function setFullPath(&$fileToUpload)
+{
+    $config           = NDB_Config::singleton();
+    $genomic_data_dir = $config->getSetting('GenomicDataPath')
+        . "/genomic_uploader/";
```

Comment:
This fails when `$config->getSetting('GenomicDataPath')` returns a path with a trailing slash.

e.g. if `$config->getSetting('GenomicDataPath')` = `/data/genomics/` then `$genomic_data_dir` will be set to `/data/genomics//genomic_uploader/file.txt`.

Currently, this only returns a PHP warning and it appears from the front-end that the file upload succeeded when it actually fails.

This should perform validation to check for trailing slashes and/or throw an exception if the path is not existent and writable.